### PR TITLE
Update Bindings: v2.0.2

### PR DESCRIPTION
**Pull request by Leeoc**
Update bindings to `v2.0.2`
API version `v1.249.4`

### CODE REVIEW

File: bindings/python/src/cloudsmith_api/models/package_dependencies.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class PackageDependencies(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'dependencies': 'list[PackageDependency]'
+        'dependencies': 'str'
```
Comment (chrisimcevoy) [inline bindings/python/src/cloudsmith_api/models/package_dependencies.py:36]: Is this kosher?

Comment (chrisimcevoy) [reply bindings/python/src/cloudsmith_api/models/package_dependencies.py:36]: I realise this wasn't what you were working on, but... might need looked into I'm afraid.

File: bindings/python/src/cloudsmith_api/models/package_tag_request.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ class PackageTagRequest(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'action': 'int',
+        'action': 'str',
```
Comment (chrisimcevoy) [inline bindings/python/src/cloudsmith_api/models/package_tag_request.py:36]: Pretty sure this is as expected... right?

File: bindings/python/src/cloudsmith_api/models/repository_geo_ip_country_code_rules.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 from cloudsmith_api.configuration import Configuration
 
 
-class InlineResponse200CountryCode(object):
+class RepositoryGeoIpCountryCodeRules(object):
```
Comment (chrisimcevoy) [inline bindings/python/src/cloudsmith_api/models/repository_geo_ip_country_code_rules.py:22]: Think this looks like an example of somewhere where we've manually touched up the contract/schema to make the generated names in the bindings more palatable. Might need to do something similar elsewhere here...?

File: bindings/python/src/cloudsmith_api/models/repository_geo_ip_rules1.py
Comment (chrisimcevoy) [inline bindings/python/src/cloudsmith_api/models/repository_geo_ip_rules1.py:1]: Another weird name 🤔

File: bindings/python/src/cloudsmith_api/models/repository_geo_ip_rules2.py
Comment (chrisimcevoy) [inline bindings/python/src/cloudsmith_api/models/repository_geo_ip_rules2.py:1]: Yet another weird name 🤔 are there any differences between this and repository_geo_ip_rules1.py?

**Review by chrisimcevoy — COMMENTED** [review]
I concentrated on the python bindings for the meantime, but better to get this feedback sooner and look at the other bindings later. I think there might be a few things to look into before we can go ahead with this - most of which aren't related to your tranche of work per se, but still need addressed. If needs be we can speak to the relevant people involved in those features.

File: bindings/python/src/cloudsmith_api/models/package_dependencies.py (hunk 1 above)
Comment (Leeoc) [reply bindings/python/src/cloudsmith_api/models/package_dependencies.py:36]: No idea why its done this, noticed a similar issue with the go bindings but a fix to the main app fixed them...

File: bindings/python/src/cloudsmith_api/models/repository_geo_ip_rules2.py
Comment (Leeoc) [reply bindings/python/src/cloudsmith_api/models/repository_geo_ip_rules2.py:1]: Seem to be the exact same, no idea why its doing that

File: bindings/python/src/cloudsmith_api/models/package_tag_request.py (hunk 2 above)
Comment (Leeoc) [reply bindings/python/src/cloudsmith_api/models/package_tag_request.py:36]: Seems right, action is an enum. Looks like it lines up with other endpoints api spec (Vulnerability Policy)

**Comment by chrisimcevoy** [comment]
I reckon the generic names for cursor-based pagination responses weren't picked up during the TF provider work simply because our TF provider doesn't use them... Will try to look at this.

**Comment by chrisimcevoy** [comment]
@Leeoc  I think all of the previous comments have been addressed. Our only issue currently is that the package dependencies endpoint response's `operator` property contains symbols which the java generator in this version of swagger-codegen doesn't know how to generate valid enum members for. Investigating.

**Pull request merged**